pkg/auditlogs/client.go

```diff
@@ -0,0 +1,280 @@
+package auditlogs
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/workos/workos-go/pkg/workos_errors"
+
+	"github.com/workos/workos-go/internal/workos"
+)
+
+// ResponseLimit is the default number of records to limit a response to.
+const ResponseLimit = 10
+
+// Order represents the order of records.
+type Order string
+
+// Constants that enumerate the available orders.
+const (
+	Asc  Order = "asc"
+	Desc Order = "desc"
+)
+
+// Client represents a client that performs auditlogs requests to WorkOS API.
+type Client struct {
+	// The WorkOS api key. It can be found in
```

Comment:
nitpick but our similar comments are inline with links (I think there's an argument for putting the links in their own section like the go docs suggest, but right now this is all just inline)

Reply:
So we discussed it on Slack and I'm following the convention from other parts of the repo where there's a limit of approx. 70-80 chars for breaking the line in field comments.